lib/main_common.pm

```diff
@@ -1548,6 +1548,7 @@ sub load_extra_tests_console {
     loadtest "console/dracut";
     loadtest 'console/timezone';
     loadtest 'console/procps';
+    loadtest 'console/autofs';
```

Comment:
Wouldn't that fit better within `load_extra_tests_filesystem`? The generic "extra_tests_console" are used as the "general dumping ground" and we should handle that carefully as the test review becomes quite hard and also frustrating. Please keep in mind for that also that any label carry over is depending on the failed modules. With any difference in which modules fail any label will not carry over.

Reply:
local opensuse and suse autofs test are now in `load_extra_tests_filesystem`